docs/cloud/pouta/connecting-to-vm.md

```diff
@@ -68,15 +68,15 @@ environments or security policies.</span>
 Before connecting to your virtual  machine, you can check its status
 in the *Instances* view of the cPouta/ePouta web interface.
 
-![VM Status check](/img/Screenshot-Instances+view+-+OpenStack+Dashboard+-+Chromium.png)
+![VM Status check](/img/pouta-instance-details.png)
 
 **Figure** The Instances view of the cPouta web interface.
 
 The figure above shows a sample of the Instances view in cPouta web
 interface. In this case, we can see that a virtual machine called
-*a-dummy-instance* is active and running. The machine has two IP
-addresses, of which the address 86.50.169.111 is the public one. The
-machine uses a keypair called *risto*.
+*test-instance-1* is active and running. The machine has two IP
+addresses, of which the address 86.50.169.245 is the public one. The
```

Comment:
This should be changed to refer to a Floating IP from pouta.csc.fi

Reply:
Hmm correct, changed!